app/dashboard/views/support.py

```diff
@@ -0,0 +1,93 @@
+import json
+import urllib.parse
+from typing import Union
+
+import requests
+from flask import render_template, request, flash, url_for, redirect
+from flask_login import login_required, current_user
+from werkzeug.datastructures import FileStorage
+
+from app.dashboard.base import dashboard_bp
+from app.log import LOG
+from app.models import Mailbox
+from app.config import ZENDESK_HOST
+
+VALID_MIME_TYPES = ['text/plain', 'message/rfc822']
+
+
+@dashboard_bp.route("/support", methods=["GET"])
+@login_required
+def show_support_dialog():
+    mailbox = Mailbox.get(current_user.default_mailbox_id)
+    return render_template("dashboard/support.html", ticketEmail=mailbox.email)
+
+
+def upload_file_to_zendesk(file: FileStorage) -> Union[None, str]:
```

Comment:
This function returns a token which isn't obvious to tell from the method name, it's maybe useful to add a small doc here.

Reply:
Updated the method name to make it clear